### PR TITLE
[Develop] feature/architecture

### PR DESCRIPTION
**Pull request by HongtingYin**
https://www.notion.so/417963db3cdf4ac0b4c36a9b72950b2c?v=88fd7654e97c46cc9c81cbd37307123e&p=3ea0302938e04b8c83edb4aa8d58640e

### CODE REVIEW

**Comment by syumet** [comment]
should be more informative, see an example [here](https://www.notion.so/0f23d2c25e624e7faf36675d3e64dd1f?v=e3f9938f22934f6b9b4ccbd4fd5f95f7&p=61d5dc8c5c6b4048b9ffec54d2ec5d67)

**Comment by HongtingYin** [comment]
Hi Steven, I've complete the file README.md. Please check it!

File: linux_sql/README.md [hunk 1]
```diff
@@ -1,2 +1,90 @@
 # Linux Cluster Monitoring Agent
-This project is under devel opment. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Review.
+
+##Introduction
```
Comment (syumet) [inline linux_sql/README.md]: ## Introduction

Comment (syumet) [reply linux_sql/README.md]: you missed one space

Comment (HongtingYin) [reply linux_sql/README.md]: Thanks! Updated.

**Pull request merged**